main.go

```diff
@@ -464,14 +485,6 @@ func RunWorker() (exitCode ExitCode) {
 		panic(err)
 	}
 
-	defer func() {
-		if r := recover(); r != nil {
-			log.Print(string(debug.Stack()))
-			log.Print(" *********** PANIC occurred! *********** ")
-			log.Printf("%v", r)
-			exitCode = INTERNAL_ERROR
-		}
-	}()
```

Comment:
As mentioned above - all the tests call `RunWorker()` so let's keep the panic handling inside of it.